src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/Transformer.kt

```diff
@@ -56,6 +63,29 @@ fun CourtCaseResponse.toMigrationDpsCourtCase() = MigrationCreateCourtCase(
     null
   },
 )
+fun CourtCaseResponse.toBookingCloneDpsCourtCase() = BookingCreateCourtCase(
```

Comment:
All these are identical to their migration counter parts - not much we can do about the duplication given the DTOs are generated from DPS OpenAPI spec - so they are just a bunch of copy and paste from the the code above in each case